webrecorder/browsertrix
# feat: App bar enhancements

Resolves https://github.com/webrecorder/browsertrix/issues/468



### Changes

- Always shows current org name
- Moves org dropdown next to logo
- Reduces logo size when logged in at smaller screen sizes

### Manual testing
1. Log in as user in multiple orgs
2. Click dropdown arrow next to org name. Verify you can select another org
3. Click org name. Verify you're taken to the org dashboard
4. Log out and log in as user in single org. Verify org name is shown next to logo
5. Resize screen to smaller width. Verify logo size is reduced

### Screenshots

#### User in single org
<img width="1512" alt="Screenshot 2024-08-06 at 12 51 05 PM" src="https://github.com/user-attachments/assets/f0859af1-0379-497c-adfb-98e7a55686d7">

#### User in multiple orgs
<img width="1512" alt="Screenshot 2024-08-06 at 12 54 19 PM" src="https://github.com/user-attachments/assets/2cec937f-bd60-4ee8-9463-0c3fe9bc374d">

#### Superadmin
<img width="1512" alt="Screenshot 2024-08-06 at 4 31 21 PM" src="https://github.com/user-attachments/assets/f3fc3a45-20e6-4252-9b3e-d8bedeccfbad">
